src/Inkluzitron/Services/TypeReaders/DateTimeTypeReader.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Discord.Commands;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace GrillBot.App.Infrastructure.TypeReaders
+{
+    public class DateTimeTypeReader : TypeReader
+    {
+        private const RegexOptions regexOptions = RegexOptions.IgnoreCase | RegexOptions.Multiline | RegexOptions.CultureInvariant | RegexOptions.IgnorePatternWhitespace;
+
+        private Dictionary<Func<Regex>, Func<DateTime>> MatchingFunctions { get; } = new Dictionary<Func<Regex>, Func<DateTime>>()
+        {
+            { () => new Regex("^(today|dnes|dneska)$", regexOptions), () => DateTime.Today }, // today, dnes, dneska
+            { () => new Regex("^(tommorow|z[i|í]tra|za[j|v]tra)$", regexOptions), () => DateTime.Now.AddDays(1) }, // tommorow, zítra, zitra, zajtra, zavtra
+            { () => new Regex("^(v[c|č]era|yesterday|vchora)$", regexOptions), () => DateTime.Now.AddDays(-1) }, // vcera, včera, yesterday, vchora
+            { () => new Regex("^(poz[i|í]t[r|ř][i|í]|pozajtra|poslezavtra)$", regexOptions), () => DateTime.Now.AddDays(2) }, // pozítří, pozitri, pozajtra, poslezavtra
+            { () => new Regex("^(te[ď|d]|now|(te|za)raz)$", regexOptions), () => DateTime.Now } // teď, ted, now, teraz, zaraz

```

Comment:
```suggestion
            { () => new Regex("^(today|dnes(ka)?)$", regexOptions), () => DateTime.Today }, // today, dnes, dneska
            { () => new Regex("^(tommorow|z[ií]tra|zajtra)$", regexOptions), () => DateTime.Now.AddDays(1) }, // tommorow, zítra, zitra, zajtra
            { () => new Regex("^(v[cč]era|yesterday)$", regexOptions), () => DateTime.Now.AddDays(-1) }, // vcera, včera, yesterday
            { () => new Regex("^(poz[ií]t[rř][ií]|pozajtra)$", regexOptions), () => DateTime.Now.AddDays(2) }, // pozítří, pozitri, pozajtra
            { () => new Regex("^(te[ďd]|now|teraz)$", regexOptions), () => DateTime.Now } // teď, ted, now, teraz
```

Also co je dpč "zavtra", "vchora", "poslezavtra" a "zaraz"? 😄

Reply:
Ukrajinština/Ruština 😄

Reply:
Tak teda ukrajinštinu/ruštinu tam nechcu 😄